src/gui_common/SpeciesDetailsPanel.cs

```diff
@@ -12,21 +11,12 @@ public class SpeciesDetailsPanel : VBoxContainer
     [Export]
     public NodePath HexPreviewPath = null!;
 
-    [Export]
-    public NodePath FossilisationButtonPath = null!;
-
-    [Export]
-    public NodePath FossilisationDialogPath = null!;
-
 #pragma warning disable CA2213
     private CustomRichTextLabel speciesDetailsLabel = null!;
-    private SpeciesPreview? speciesPreview;
+    private SpeciesPreview speciesPreview = null!;
     private CellHexesPreview hexesPreview = null!;
-    private Button fossilisationButton = null!;
-    private FossilisationDialog fossilisationDialog = null!;
-#pragma warning restore CA2213
-
     private Species? previewSpecies;
+#pragma warning restore CA2213
```

Comment:
This pragma needs to be before `previewSpecies` variable as the pragma is to be only used around Godot Node variables, so it being around this species variable is wrong.